tensorboard/plugins/profile/tf_op_profile/utils.ts

```diff
@@ -36,7 +36,7 @@ export function flameColor(
     rgba(2 * (1 - fraction) * brightness, brightness, 0, opacity);
 }
 
-export function utilization(node) {
+export function utilization(node: any) {
```

Comment:
Why is `any` needed here?

Reply:
In google3, if building using ts_library. The compiler requires adding the :any explicitly. Therefore added here to make it easier to build internally into a vulcanized html.

Reply:
OK!